astropy/io/fits/verify.py

```diff
@@ -1,4 +1,4 @@
-# Licensed under a 3-clause BSD style license - see PYFITS.rst
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Hmm, there is still `PYFITS.rst` license in the list of distributed licenses though.